modules/whs-vrkit/src/lib/WebVR.js

```diff
@@ -0,0 +1,186 @@
+/**
```

Comment:
@hirako2000 No need. @mrdoob says VR API in Three.js will be changed soon. No more even `VREffect` & `VRControls`